perf/perf_rawevents.py

```diff
@@ -98,6 +99,13 @@ def test(self):
         elif cpu_family == 'power9':
             self.run_event('raw_codes_p9', 'raw')
             self.error_check()
+            self.run_event('name_events_p9', 'name')
+            self.error_check()
+        elif cpu_family == 'power10':
+            self.run_event('raw_codes_p10', 'raw')
+            self.error_check()
+            self.run_event('name_events_p10', 'name')
+            self.error_check()
```

Comment:
Hi @disgoel ,
can we make this block more generic? so that its not required to add one more elif for power11.
somethings like as below,

cpu_family = cpu.get_family()[5:]
events = ['raw', 'name']
for event in events:
     if event == 'raw':
          file_name = event + '_codes_p' + cpu_family
          perf_flags = "perf stat -e r"
     elif eve == 'name':
          file_name = event + '_names_p' + cpu_family
          perf_flags = "perf stat -e "
     self.run_event(file_name, perf_flags)
     self.error_check()

Note : you should remove the following code (line 75-78)
        if eventname == 'raw':
            perf_flags = "perf stat -e r"
        elif eventname == 'name':
            perf_flags = "perf stat -e " 

And 
your     def run_event(self, filename, eventname): will be as below
             def run_event(self, filename, perf_flags):
or, you can use the other better names too if you wish to. but u need to change other lines of code too, whereever this name exists.